tests/test_project/viewtest/tests.py

```diff
@@ -438,3 +439,61 @@ def test_no_schema_list(self):
         where_fragment, params = _make_where(schemaname=None, tablename=["test_tablename1", "test_tablename2"])
         self.assertEqual(where_fragment, "tablename IN (%s, %s)")
         self.assertEqual(params, ["test_tablename1", "test_tablename2"])
+
+
+class TestMaterializedViewSyncDisabledSettings(TestCase):
+    def setUp(self):
+        """
+        NOTE: By default, Django runs and registers signals with default values during
+        test execution. To address this, we store the original receivers and settings,
+        then restore them in tearDown to avoid affecting other tests.
+        """
+        from django.db.models.signals import post_migrate
```

Comment:
Consider moving the imports to the top, unless there's a reason for it (circular import etc.)